ansible_collections/arista/avd/docs/porting-guides/5.x.x.md

```diff
@@ -1502,13 +1531,13 @@ router_bgp:
 
 The change has been incorporated into `traffic-policies.j2` eos template.
 
-In AVD 4.0.0, configuring the `flags` for `UDP` protocol under traffic-policies was allowed and the configuration for `source/destination port` and `source/destination field-set` together was not generated for `TCP/UDP` protocol.
+In AVD 4.0.0, configuring the `flags` for the `UDP` protocol under traffic policies was allowed, but the configuration for `source/destination port` and `source/destination field-set` together was not generated for the `TCP/UDP` protocols.
 
-As of AVD 5.0.0, the configuration of `flags` in `UDP` protocol under traffic-policies is not generated and the configuration for `source/destination port` and `field set` can be generated together for `TCP/UDP` protocol. Also, the configuration order has been fixed as per the EOS running-config.
+As of AVD 5.0.0, the configuration of `flags` in the `UDP` protocol under traffic policies is not generated, and the configuration for `source/destination port` and `field set` can be generated together for the `TCP/UDP` protocol. Also, the configuration order has been fixed as per the EOS running-config.
```

Comment:
@carlbuchmann a bit confused here. Should the wording be "is now generated"?

Reply:
good catch, resolved!